scaaml/capture/aes/communication.py

```diff
@@ -1,10 +1,14 @@
 """The target in cw."""
 import chipwhisperer as cw
 
+from scaaml.capture.communication import AbstractSCommunication
 
-class SCommunication:
+
+class SCommunication(AbstractSCommunication):
```

Comment:
With the protocol version and the mention of a scope, this means that this communication implementation is meant to use a Chipwhisperer.
For other captures where OpenADC (the scope embedded in the Chipwhisperer boards) cannot be used, I usually really on an external UART.

Maybe this class should be named `ChipwhispererComm` or `CWCommunication`?